packages/app/src/systems/Core/__tests__/utils/reparse.ts

```diff
@@ -0,0 +1,5 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
+
+export function reparse(v: any) {
```

Comment:
Do we need this? I didn't see this being used in other places.

Reply:
some times comparting objects can be tricky if you have `BN` instances in it.

this function allows us to strict compare objects even with BN instances, increasing coverage because we can still use toStrictEqual `expect(tx).toStrictEqual(tx2)`

Reply:
@luizstacio I added this function to core/utils..

this place was also doing same thing. -> packages/app/src/systems/Transaction/utils/transaction.tsx

replace to use `reparse` as well